en/modules/modules_driver.md

```diff
@@ -84,7 +84,7 @@ Source: [drivers/batt_smbus](https://github.com/PX4/Firmware/tree/master/src/dri
 
```

Comment:
@nbelanger99 Please undo the changes in this document. This is auto generated from the in-source comments in PX4-Autopilot. You can propose changes in the source separately.

Reply:
Ah, I see, thank you. This is not true of the other files changed in this PR, correct?

Reply:
Correct, this is the only one in source code.